tests/osaka/eip7692_eof_v1/eip7480_data_section/test_data_opcodes.py

```diff
@@ -14,6 +15,24 @@
 pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
 
 
+@pytest.mark.parametrize("index", [0, 1, 31, 32, 33, 63, 64])
+@pytest.mark.parametrize("suffix_len", [0, 1, 31, 32, 24000])
+def test_dataloadn(eof_state_test: EOFStateTestFiller, index: int, suffix_len: int):
+    """Basic tests for DATALOADN execution."""
+    sentinel = 0x8000000000000000000000000000000000000000000000000000000000000001
+    eof_state_test(
+        container=Container(
+            sections=[
+                Section.Code(
+                    Op.DATALOADN[index] + Op.SSTORE(0, unchecked=True) + Op.STOP,
```

Comment:
```suggestion
                    Op.SSTORE(0, Op.DATALOADN[index]) + Op.STOP,
```
Or is the current arrangement used to keep the usage of `Op.DATALOADN` at the start of the line?

Reply:
I'm not sure why it happened :/